concent_api/conductor/models.py

```diff
@@ -0,0 +1,77 @@
+from django.core.validators import ValidationError
+from django.db.models       import CharField
+from django.db.models       import DateTimeField
+from django.db.models       import ForeignKey
+from django.db.models       import Model
+from django.db.models       import OneToOneField
+from django.utils           import timezone
+
+from core.constants         import MESSAGE_TASK_ID_MAX_LENGTH
+from utils.fields           import ChoiceEnum
+from .constants             import MESSAGE_PATH_LENGTH
+
+
+class VerificationRequest(Model):
+
+    subtask_id = CharField(max_length = MESSAGE_TASK_ID_MAX_LENGTH)
+
+    # Relative path of the .zip file that contains Blender source files for the render.
+    source_package_path = CharField(max_length=255, unique=True)
+
+    # Relative path of the .zip file that contains the rendering result received from the provider.
+    result_package_path = CharField(max_length=255, unique=True)
+
+    def clean(self):
+        super().clean()
+
+        # source_package_path cannot be used as result_package_path in any model instance.
+        if VerificationRequest.objects.filter(result_package_path=self.source_package_path).exists():
+            raise ValidationError({
+                'source_package_path': 'source_package_path cannot be used as result_package_path in any other VerificationRequest.'
+            })
+
+        # result_package_path cannot be used as source_package_path in any model instance.
+        if VerificationRequest.objects.filter(source_package_path=self.result_package_path).exists():
+            raise ValidationError({
+                'result_package_path': 'result_package_path cannot be used as source_package_path in any other VerificationRequest.'
+            })
+
+
+class BlenderSubtaskDefinition(Model):
+    """
+    For each VerificationRequest there must be exactly one BlenderSubtaskDefinition in the database.
+    """
+
+    class OutputFormat(ChoiceEnum):
```

Comment:
Are we supposed to support just those two formats? In the [blender docs]( https://docs.blender.org/manual/en/dev/data_system/files/media/image_formats.html#image-formats) seems to be more of it ;)

Reply:
Off course we should, I was looking for this but couldn't find it, thanks for the link !

Reply:
Let's drop EXR.

Also, please make sure that both uppercase and lowercase versions are supported.

Reply:
done :+1: 

What do you mean by `are supported` ?

Reply:
I mean that `TaskToCompute` can specify the format as either `PNG` or `png` and both are accepted by Golem (see https://github.com/golemfactory/golem/issues/218). And maybe things like `PnG` are accepted too. We should be able to handle all values accepted by Golem.

Though not necessarily in this enum - we could assume that Concent e.g. converts extension to lower case when it passes the format in a Celery task.

Reply:
:+1: Off course, but that will be handled in the following pull requests, here is only definition.